doc/dev/testing.rst

```diff
@@ -32,7 +32,7 @@ Using
 
 #. Smoke test::
 
-     make && ctest -j12
+     ninja && ctest -j12
```

Comment:
@ljflores hi Laura, would you kindly update https://github.com/ceph/ceph/blob/6824b29ca1d843cbac81ca2d7caef14640b21826/src/script/build-integration-branch#L112 
accordingly ?

or, could take this opportunity to replace `make && ctest -j12` with `./run-make-check.sh`. as `ninja` alone does not prepare the build dependencies or run `cmake`.

Reply:
Yes of course!